### PR TITLE
add babel loader

### PR DESCRIPTION
**Pull request by ShvetaJain**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,5 +5,13 @@ module.exports = {
...
+  },
+  module: {
+      rules: [{
+          test: [/\.js$/],
+          exclude: /node_modules/,
+          loaders: ['babel-loader'],
+        }
+      ],
+    },
 };
```
Comment (abp437) [inline webpack.config.js]: Always add an extra line at the end of file.

File: app.js [hunk 2]
```diff
@@ -1 +1 @@
-const test = "vishal teacher";
\ No newline at end of file
+const test = "vishal teacher";
```
Comment (abp437) [inline app.js]: Single quotes in js file.

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+"presets": ["es2015"]
```
Comment (abp437) [inline .babelrc]: Don't use this preset, instead use `env`, it has much wider support.

File: webpack.config.js [hunk 4]
```diff
@@ -5,5 +5,13 @@ module.exports = {
   output: {
     filename: 'bundle.js',
     path: path.resolve(__dirname, 'dist')
-  }
+  },
+  module: {
+      rules: [{
+          test: [/\.js$/],
```
Comment (abp437) [inline webpack.config.js]: Add support even for `.jsx` files.

**Review by abp437 — CHANGES_REQUESTED** [review]
Need Changes.

File: webpack.config.js [hunk 5]
```diff
@@ -5,5 +5,13 @@ module.exports = {
...
+          exclude: /node_modules/,
+          loaders: ['babel-loader'],
+        }
+      ],
+    },
```
Comment (visha12l) [inline webpack.config.js]: 1)add eslint loader 
2)use resolve from webpack https://webpack.js.org/configuration/resolve/
3)integration with codacy (its an hosted automated code review service) https://www.codacy.com/

File: src/app.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+const es5 = function () {
+  function test1() {
```
Comment (visha12l) [inline src/app.js]: use es6 arrow functions

File: src/app.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    const test = 'jain',
+      age = 80;
+    return `shveta ${test} ${age}`;
+  }
+
+  function objectTest() {
```
Comment (visha12l) [inline src/app.js]: use es6 arrow functions

File: src/app.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    const url = 'test/test.html',
+      object = {
+        url,
+      };
+  }
+
+  function destructure() {
```
Comment (visha12l) [inline src/app.js]: use es6 arrow functions

**Pull request merged**